docs/03-module-block-3/02-learning-goals.adoc

```diff
@@ -31,20 +31,20 @@ tbd.
   methods.
 
 [[LG-3-2]]
-==== LG 3-2: Relevant Qualities
+==== LG 3-2: Know relevant qualities
 
 * Know the qualities that typically indicate the use of formal
   methods, such as reliability, safety, and security.
```

Comment:
Are these qualities of ISO25010? Then you should mention it here.

Reply:
They are, but I disagree - ISO 25010 has no special status in Foundation, either, and mentioning ISO 25010 would add no useful information.